number/serializer/9b8b959e-3c8c-4e4c-ac8a-7bf59bb50803.json

```diff
@@ -9,19 +9,28 @@
   "interface": {
     "inputs": {
       "607be551-0389-42f8-99bb-088bfd8b1579": {
-        "type": "any",
-        "name": "number"
+        "type": "number",
+        "name": "number",
+        "description": "The number to be converted."
       }
     },
     "outputs": {
       "60eb7bd0-fd76-48a5-a787-78a6f87d131b": {
-        "type": "any",
-        "name": "string"
+        "type": "string",
+        "name": "string",
+        "description": "Outputs the serialized number."
       }
     }
   },
   "name": "number/Serializer",
   "attributes": {
     "locked": true
+  },
+  "description": "Serializes the given number to a string.\n\nExample:\n\n1. 23.45@0 is received via `number`\n2. \"23.45\"@0 is sent via `string`",
```

Comment:
Is "serialize" to most proper term here? I'd rather call this node `number/To string` or `number/Stringify` or `number/